README.md

```diff
@@ -76,8 +76,8 @@ The following information should additionally be available in the witness:
 
 | key | Meaning | Allowed in Violation Witnesses | Allowed in Correctness Witnesses |
 | --- | --- | ---- | ---- |
-| threadId | Represents the currently active thread for the transition. If no ``threadId`` is given, any thread can be active. The value should be a unique identifer for a thread. | Yes | Yes |
-| createThread | The currently active thread (value of ``threadId``) creates a new thread (value of ``createThread``) . In general, using a ``threadId`` is only allowed after creating a matching thread. The new thread's function can be entered on a second transition following this transition, such that the transition with the ``enterFunction`` key has the ``threadId`` of the created thread. When the function of the thread is exited, the thread is assumed to be terminated and its ``threadId`` should no longer be used. | Yes | Yes |
+| threadId | ``createThread`` is used in the analysis of concurrent programs as part of the source-code guard of a transition and restricts the set of CFA edges matched by the source-code guard to operations where a new thread is created. The value of ``data`` elements with this key is an identifier for the new thread. Any string may be used as an identifier, provided that it is unique in the document in the sense that for any run through the automaton, there may not be two different threads that (1) are both active (i.e., created but not yet destroyed) and (2) share the same identifier. The ``attr.type`` attribute of this key is ``string``. | Yes | Yes |
```

Comment:
this needs review by @kfriedberger and @hernanponcedeleon. There are currently pull requests for changing/fixing the underspecification for concurrency witnesses, cf. #30 #34. Please coordinate and tell me which version should be taken.
If this gets controversial I would recommend to take this section out and merge it separately, such that we can more easily merge the rest of this PR which is without conflict.

Reply:
- I don't think #30 is related --- regarding the content, I mean; of course both change sets need to be merged manually. By the way, I don't really see the point of #30 anyway, because we already allow an initial transition with enterFunction(main); the createThread for the main thread belongs on that transition, too, in my opinion. But that's off-topic here.
- I think the improvements proposed in #34 are very valuable. Regarding the wording, I will discuss some modification with @kfriedberger at the soonest opportunity.

Reply:
Was there any further discussion yet?
From my point the new description looks valid and sufficient.